webpack/ForemanInventoryUpload/Components/AutoUploadSwitcher/AutoUploadSwitcherActions.js

```diff
@@ -8,7 +8,7 @@ export const handleToggle = currentAutoUploadEnabled => async dispatch => {
   try {
     const {
       data: { autoUploadEnabled },
-    } = await API.post(inventoryUrl('auto_upload'), {
+    } = await API.post(inventoryUrl('enable_cloud_connector'), {
```

Comment:
Ignore this, will need to be reverted. This was simply to make testing for me easier at the start.